lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -531,10 +536,19 @@ private static function get_nav_element_directives( $should_load_view_script ) {
 			),
 			JSON_HEX_TAG | JSON_HEX_APOS | JSON_HEX_AMP
 		);
-		return '
+		$nav_element_directives = '
 			data-wp-interactive=\'{"namespace":"core/navigation"}\'
 			data-wp-context=\'' . $nav_element_context . '\'
 		';
+
+		// When the navigation overlayMenu attribute is set to "always"
+		// we don't need to use JavaScript to collapse the menu as we set the class manually.
+		if ( ! static::is_always_overlay( $attributes ) ) {
+			$nav_element_directives .= 'data-wp-init="callbacks.initNav" ';
+			$nav_element_directives .= 'data-wp-class--is-collapsed="context.isCollapsed"';
```

Comment:
```suggestion
			$nav_element_directives .= 'data-wp-init="callbacks.initNav"';
			$nav_element_directives .= ' '; // space separator
			$nav_element_directives .= 'data-wp-class--is-collapsed="context.isCollapsed"';
```

Just an idea...might be open to typos otherwise.